upgrade.go

```diff
@@ -65,6 +65,9 @@ func (u upSlice) Print(start int) {
 		var left, right string
 
 		f := func(name string) (output string) {
```

Comment:
This should probably be moved to its own function, all other checks to `if alpmConf.Options&alpm.ConfColor == 0` have their own function.

Reply:
To clarify because it's already its own function technically. I meant move it to it's no long inside Print and give it a decent name.

Reply:
Sure, I agree. But this is a bug fix, so I wanted to keep the patch small. There's no reason to do a refactor in the same commit.

Reply:
You have a point, it was already like this all you did was add the missing check. Change is small and good so I'll add it to the next realise. We can refactor during the feature freeze.